doc/user-manual/language/cubical.lagda.rst

```diff
@@ -3,79 +3,117 @@
+decomposed into homogeneous composition and generalized
+transport. This is what makes the general schema for higher inductive
+types work, following the `CHM`_ paper.
 
-To use cubical type theory, you need to run Agda with the ``--cubical`` command-line-option.
-You can also write ``{-# OPTIONS --cubical #-}`` at the top of the file.
+To use the cubical mode Agda needs to be run with the ``--cubical``
+command-line-option or with ``{-# OPTIONS --cubical #-}`` at the top
+of the file.
 
-To define paths, use λ abstractions. For example, this is the definition of the constant path:
+The cubical mode adds the following features to Agda:
 
-..
-  ::
-  module refl-example where
+1. An interval ``I`` type and path types
+2. Partial elements and systems
+3. Kan operations (``transp`` and ``hcomp``)
+4. ``Glue`` types
+5. Higher inductive types
+6. Cubical identity types
 
-::
+There is a standard library for Cubical Agda available at
+https://github.com/agda/cubical.
 
-    refl : ∀ {a} {A : Set a} {x : A} → Path x x
-    refl {x = x} = λ i → x
+The main design choices of the core part of the library are explained
+in https://homotopytypetheory.org/2018/12/06/cubical-agda/.
 
-Although they use the same syntax, a path is not a function.
-For example, the following is not valid:
+In order to get access to the Cubical Agda primitives one should
+either import
+https://github.com/agda/cubical/blob/master/Cubical/Core/Primitives.agda
+or add the relevant import statements from the top of that file.
 
-.. code-block:: agda
+There is also an older version of the library available at
+https://github.com/Saizan/cubical-demo/.
 
-  refl : ∀ {a} {A : Set a} {x : A} → Path x x
-  refl {x = x} = λ (i : _) → x
+The interval and path types
+---------------------------
 
--------------------------
-The interval type (``I``)
--------------------------
+The key idea of Cubical Type Theory is to add an interval type ``I :
+Setω`` (the reason this is in ``Setω`` is because it doesn't support
+the Kan operations). A variable ``i : I`` intuitively corresponds to a
+point the `real unit interval
+<https://en.wikipedia.org/wiki/Unit_interval>`_. In an empty context,
+there are only two values of type ``I``: the two endpoints of the
+interval, ``i0`` and ``i1``.
 
-The variable ``i`` in the definition of ``refl`` has type ``I``, which
-topologically corresponds to the `real unit interval <https://en.wikipedia.org/wiki/Unit_interval>`_.
-In a closed context, there are only two values of type ``I``: the two
-ends of the interval, ``i0`` and ``i1``::
+.. code-block:: agda
 
-  a b : I
-  a = i0
-  b = i1
+  i0 : I
+  i1 : I
 
-Elements of the interval form a `De Morgan algebra <https://en.wikipedia.org/wiki/De_Morgan_algebra>`_,
-with minimum (``∧``), maximum (``∨``) and negation (``~``).
+Elements of the interval form a `De Morgan algebra
+<https://en.wikipedia.org/wiki/De_Morgan_algebra>`_, with minimum
+(``∧``), maximum (``∨``) and negation (``~``).
 
```

Comment:
It should be made clear that various examples use notations from [agda/cubical](https://github.com/agda/cubical)  (for instance `\/` and `/\`), so users know that they may have to download and setup an additional library to try them. For the people who would not want to use [agda/cubical](https://github.com/agda/cubical), the names of all the primitives should be mentioned somehow (maybe in parenthetical remarks when they are introduced, or in a separate section.

Reply:
I added an appendix listing all of the primitives and built-ins.